### PR TITLE
Alignment software manual tracking and tracking refactoring

### PR DESCRIPTION
**Pull request by luctowers**
Ready for review.
- manual tracking window
- lots of refactoring in auto tracking
- status indicator icons for both manual and auto tracking

### CODE REVIEW

File: nrcemt/alignment_software/gui/manual_track/frame_particle_adjustment.py [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+
+class ParticleAdjustmentFrame(tk.LabelFrame):
+
+    def __init__(
+        self, master, particle_count,
+        select_command=None, interpolate_command=None, move_command=None,
+        delete_command=None, reset_command=None
+    ):
+        super().__init__(master, text="Particle selection and ajustment", bd=1)
```
Comment (coffeehousejazz) [inline nrcemt/alignment_software/gui/manual_track/frame_particle_adjustment.py:17]: spelling error - adjustment

**Review by coffeehousejazz — COMMENTED** [review]
looking good - need more comments but I suspect that will be added in another pr

**Comment by luctowers** [comment]
> looking good - need more comments but I suspect that will be added in another pr

yeah :/ I was gonna do a big pr with a ton of docstrings

**Comment by coffeehousejazz** [comment]
> > looking good - need more comments but I suspect that will be added in another pr
> 
> yeah :/ I was gonna do a big pr with a ton of docstrings

good idea !

**Review by coffeehousejazz — APPROVED** [review]
looking nice

**Pull request merged**